poo/src/main/java/edu/jesus/gaspar/actividades/actividad_1/models/Car.java

```diff
@@ -6,30 +6,34 @@ public class Car {
     private int numeroPuertas;
     private String color;
 
-    @Override
-    public String toString() {
-        return "Car{" +
-                "modelo='" + modelo + '\'' +
-                ", marca='" + marca + '\'' +
-                ", numeroPuertas=" + numeroPuertas +
-                ", color='" + color + '\'' +
-                '}';
-    }
+    /**
+     * Se genera un nuevo objeto con la info de un carro
+     * @param modelo año del carro
+     * @param marca empresa del carro
+     * @param numeroPuertas
+     * @param color variedad de colores
+    */
 
     public Car(String modelo, String marca, int numeroPuertas, String color) {
         this.modelo = modelo;
         this.marca = marca;
```

Comment:
reemplazar cada asignacion de la forma "this.variable = variable" por su setter correspondiente: "setVariable(variable)"